ssl/ssl_test.cc

```diff
@@ -5552,6 +5552,52 @@ TEST_P(SSLVersionTest, SSLPending) {
   EXPECT_EQ(1, SSL_has_pending(client_.get()));
 }
 
+// Identical test to the |SSLPending| test suite above, but with
+// |SSL_(read/peek/write)_ex| operations instead.
+TEST_P(SSLVersionTest, SSLPendingEx) {
+  UniquePtr<SSL> ssl(SSL_new(client_ctx_.get()));
+  ASSERT_TRUE(ssl);
+  EXPECT_EQ(0, SSL_pending(ssl.get()));
+
+  ASSERT_TRUE(Connect());
+  EXPECT_EQ(0, SSL_pending(client_.get()));
+  EXPECT_EQ(0, SSL_has_pending(client_.get()));
+
+  size_t buf_len;
+  ASSERT_TRUE(SSL_write_ex(server_.get(), "hello", 5, &buf_len));
```

Comment:
```suggestion
  ASSERT_EQUAL(SSL_write_ex(server_.get(), "hello", 5, &buf_len), 1);
```
Would return 5 still count as "true" and pass this test? You do that down below for the read/peek tests.

Reply:
Yeah my bad, I was planning on this to be consistent.